complaint_search/es_interface.py

```diff
@@ -61,10 +57,8 @@ def _create_and_append_bool_should_clauses(es_field_name, value_list,
 
             filter_list.append({"bool": {"should": f_list}})
 
-
-# def search(fmt="json"):
 def search(fmt="json", field="what_happened", size=10, frm=0, 
```

Comment:
That's a lot of arguments!

I would think you could use `**kwargs` except for the values you actually want to default -or-

```python
defaults = {...}
composed = defaults.update(**kwargs)
```